test/system/252-quadlet.bats

```diff
@@ -558,4 +558,28 @@ EOF
     remove_secret $SECRET_NAME
 }
 
+@test "quadlet - volume path using specifier" {
+    local tmp_path=$(mktemp -d --tmpdir quadlet.volume.XXXXXX)
+    local tmp_dir=$(basename "$tmp_path")
```

Comment:
I'm having a hard time understanding the point of doing this instead of using `$PODMAN_TMPDIR` https://github.com/containers/podman/blob/30619bba68ea6bd31d7e67a6e16e3612b6bc8760/test/system/helpers.bash#L91-L96

I'm also finding it impossible to understand the `systemd.unit(5)` documentation for `%T`:
>This is either /tmp or the path "$TMPDIR", "$TEMP" or "$TMP" are set to.

...because there is no indication of the process space in which those envariables are checked. 

Anyhow. Please use `$PODMAN_TMPDIR` or provide an explanation as to why that's not possible.

Reply:
According to the line you've added, the base directory for `PODMAN_TMPDIR` can be overridden by setting `BATS_TMPDIR`. So, if the latter is set, the directory will not be created under `/tmp` which will cause `systemd` to point to a non-existing path. For this reason I wanted to ensure the location of the directory. But, if it is never really set, I can use `PODMAN_TMPDIR`
Having said that, the idea behind the test is to check the support for the systemd specifiers. I chose to use `%T`, but I don't mind using a different one (https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Specifiers), if you think it makes more sense.
Please let me know what you think

Reply:
Done